kuma/landing/jinja2/landing/homepage.html

```diff
@@ -88,17 +88,22 @@ <h2 class="offscreen">{{ _('Web Technologies') }}</h2>
           <li><a href="{{ wiki_url('Learn/JavaScript/') }}">{{ _('Learn JavaScript') }}</a></li>
         </ul>
       </div>
-      <div class="column-home-features">
+      <div class="column-feature">
         <h3><i aria-hidden="true" class="icon-wrench"></i><span>{{ _('Developer<br />Tools') }}</span></h3>
         <ul>
-        <li><a href="{{ wiki_url('Tools/Page_Inspector') }}">{{ _('Page Inspector') }}</a></li>
-        <li><a href="{{ wiki_url('Tools/Web_Console') }}">{{ _('Web Console') }}</a></li>
-        <li><a href="{{ wiki_url('Tools/Debugger') }}">{{ _('JavaScript Debugger') }}</a></li>
-        <li><a href="{{ wiki_url('Tools/Performance') }}">{{ _('Performance Tools') }}</a></li>
-        <li><a href="{{ wiki_url('Tools') }}">{{ _('more...') }}</a></li>
+         <li><a href="{{ wiki_url('Tools/Page_Inspector') }}">{{ _('Page Inspector') }}</a></li>
+         <li><a href="{{ wiki_url('Tools/Web_Console') }}">{{ _('Web Console') }}</a></li>
+         <li><a href="{{ wiki_url('Tools/Debugger') }}">{{ _('JavaScript Debugger') }}</a></li>
+         <li><a href="{{ wiki_url('Tools/Performance') }}">{{ _('Performance Tools') }}</a></li>
+         <li><a href="{{ wiki_url('Tools') }}">{{ _('more...') }}</a></li>
```

Comment:
The ghost of @jezdez requests you use the unicode ellipsis character "…" or ``&hellip;``